ta/arch/arm/link.mk

```diff
@@ -33,7 +33,8 @@ link-script-cppflags-$(sm) := -DASM=1 \
 
 -include $(link-script-dep$(sm))
 
-$(link-script-pp$(sm)): $(link-script$(sm)) $(conf-file) $(MAKEFILE_LIST)
+link-script-pp-makefiles$(sm) = $(filter-out %.d %.cmd,$(MAKEFILE_LIST))
+$(link-script-pp$(sm)): $(link-script$(sm)) $(conf-file) $(link-script-pp-makefiles$(sm))
```

Comment:
I wonder if it wouldn't be better to use the approach in `mk/compile.mk` instead. Perhaps by using a `pp-srcs-y` variable, it's more complicated, but it could be reused for all link files that need preprocessing, on the other hand it could be overkill. What do you think?

Reply:
I don't see what you mean... How/where would you use `pp-srcs-y`? From `sub.mk`?

Reply:
Yes, from `sub.mk` . We can take this now and I'll create a PR later with `pp-srcs-y`.

Reply:
Fine with me. Thanks.